Snapper/Snapper.py

```diff
@@ -0,0 +1,36 @@
+class Snapper():
+    def __init__(self):
+        self.sensors = []
+        self.snapshot = []
+        self.startTime = 0.0
+        self.variables = []
+
+    def step(self):
+        '''
+        runs next incremental step in building a snapshot.
+        :return:
+        '''
+        pass
+
+    def onData(self, sensor, data):
+        '''
+        This function receives data from a sensor and stores it for use in snapshot building.
+        :param sensor:
+        :param data:
+        :return:
+        '''
```

Comment:
Done. We can cancel this pull request and I'll make a new one with those changes added.